### PR TITLE
Goc december more e2e

### PR DESCRIPTION
**Pull request by smarshall-spitzbart**
draft PR to stress test a lot of recv slash packets in a single block, can close later

### CODE REVIEW

File: tests/e2e/throttle.go [hunk 1]
```diff
@@ -341,57 +423,6 @@ func (s *CCVTestSuite) TestSlashMeterAllowanceChanges() {
 
 }
 
-// TestSlashSameValidator tests the edge case that that the total slashed validator power
-// queued up for a single block exceeds the slash meter allowance,
-// but some of the slash packets are for the same validator, and should all be handled.
-func (s *CCVTestSuite) TestSlashSameValidator() {
```
Comment (smarshall-spitzbart) [inline tests/e2e/throttle.go:347]: Not relevant without cherry picking #560 into this branch

**Comment by smarshall-spitzbart** [comment]
variation of this test will be added to base branch
